src/other/Table.js

```diff
@@ -23,11 +23,12 @@
 
     Table.prototype.publish("renderHtmlDataCells", false, "boolean", "enable or disable HTML within cells",null,{tags:["Private"]});
     Table.prototype.publish("pagination", false, "boolean", "Enable or disable pagination",null,{tags:["Private"]});
+    Table.prototype.publish("paginationLimit", null, "number", "Maximum number of rows allowed before pagination defaults to on",null,{tags:["Private"]});
```

Comment:
@jbrundage Interesting, there is a itemsPerPage() param but it doesn't appear to do anything in dermatology.  Will look into that.

I personally do not like the idea of defaulting to pagination... I have not seen a lot of use cases, but what I have seen suggests that most tables are displayed in rather small containers (e.g. Legend) and the pagination controls are just too clunky.